tasks/inprove-the-UI-of-the-game.md

```diff
@@ -0,0 +1,28 @@
+<!-- https://codein.withgoogle.com/dashboard/tasks/4946505678979072/ -->
+
+---
```

Comment:
is this required?

Reply:
i don't currently have the link. so just commented it out from the sample.

Reply:
Not the link the `---`

Reply:
The --- is the divider I added so I know what to copy. Its nice to have.

Reply:
oh am sorry